test/fixtures/invite_fixtures.js

```diff
@@ -8,12 +8,31 @@ const pactBase = require('./pact_base')
 // Global setup
 const pactInvites = pactBase()
 
+function buildInviteWithDefaults (opts) {
+  const data = _.defaults(opts, {
```

Comment:
what does `_.defaults` do?

Reply:
it constructs an object using the values for the keys in `opts` if they exist, otherwise uses the  values in the second object given. https://lodash.com/docs/4.17.15#defaults